assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Footer.js

```diff
@@ -28,14 +28,6 @@ import { Grid, Cell, Row } from '../../../../../material-components';
 import Pagination from './Pagination';
 
 export default function Footer( { tab, footerText } ) {
-	if ( ! footerText ) {
-		return (
-			<div className="googlesitekit-idea-hub__footer">
-				<Pagination tab={ tab } />
-			</div>
-		);
-	}
```

Comment:
Kind of a nit-pick, but is there no way to solve this while still not rendering an empty div if `footerText` is empty? We could still use the `Row` to maintain the same layout but then only render the `Cell` for the pagination - I think it's possible to use specific classes to horizontally offset?

Reply:
@felixarntz There probably is a way, but I don't think it's worth the effort to avoid rendering a single cell with empty content. It sounds like a micro optimization for this specific case, where it might even be more performant to keep the structure there for a smaller update when switching tabs (when there is `footerText`). It's simpler this way so I don't think we should worry about it as it is now.